jmri_versions.md

```diff
@@ -0,0 +1,47 @@
+# What is an API?
+
+An API (Application Programming Interface) is a set of interfaces, classes, methods and fields that can be used by software that is using the JMRI library. An example of such software is [CATS (Computer Automated Traffic System)](http://cats4ctc.wikidot.com/).
+
+The API is an offer to other developers, saying that if you want to use jmri.jar in your project, you can use these interfaces/classes/methods/fields under these conditions, for example that a particular method is available, but may change in the next minor version, while another method is granted to stay the same at least until next major version.
```

Comment:
That should start “The API is a contract” the “to” that follows could possibly be “with”, but that might require further editing.... let me think about that further...

Reply:
I have added a line at the top about the first version this document is valid for. I have also updated the text about the API.